src/notification/notification.js

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import React from 'react';
+import type {NotifcationPropsT} from './types';
+import {KIND} from './constants';
+import {StyledRoot} from './styled-components';
+import {getOverrides} from '../helpers/overrides';
+
+function Notification({children, kind, overrides = {}}: NotifcationPropsT) {
+  const [Root, baseRootProps] = getOverrides(overrides.Root, StyledRoot);
+
+  return (
+    <Root role="alert" $kind={kind} {...baseRootProps}>
```

Comment:
they say that not all notifications should have `alert` role as it is considered more intense. Think for notification not error (or even warning) we shouldn't use this role

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

Reply:
Maybe I'm miss-reading the docs, but `error` and `warning` are specifically shown using the role. Could you link to an example where a warning/error shouldn't have that role?

These should be used sparingly by design/dev, but if a notification is shown it should probably use the assertive tone to be read, right?

Reply:
Error and warning's are good. But as per design we have `primary` and `success` notifications. They don't look like `assertive`

Reply:
The `assertive` value is specifically a `POLITENESS_SETTING` and has not direction relation to what is actually being displayed.

It only means that the user will be immediately told of the notification, which should probably occur if a success notification is triggered.

Reply:
ok, makes sense